.github/workflows/ci.yaml

```diff
@@ -0,0 +1,135 @@
+name: ci
+
+on:
+  push:
+  pull_request:
+
+env:
+  BAZEL_CXXOPTS: -std=c++14
```

Comment:
I'm assuming we'll add more language versions in the future--I'd leave a TODO here.

Reply:
This replicates the current build. I'll expand this in the future.